solidity/contracts/bridge/WalletCoordinator.sol

```diff
@@ -0,0 +1,182 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity 0.8.17;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+import "./Bridge.sol";
+import "./Deposit.sol";
+import "./Wallets.sol";
+
+// TODO: Documentation and unit tests.
+// TODO: Should we make it pausable?
```

Comment:
Nope. Let's not add any pause mechanisms.

Reply:
Removed the TODO: a04cebe4